tests/test_netsuite/test_models.py

```
@@ -340,11 +340,12 @@ def test_create_credit_card_charge(db):
     expense_group = ExpenseGroup.objects.get(id=4)
     credit_card = CreditCardCharge.create_credit_card_charge(expense_group)
     configuration = Configuration.objects.get(workspace_id=2)
-    credit_card_charge_lineitem = CreditCardChargeLineItem.create_credit_card_charge_lineitem(expense_group, configuration)
+    credit_card_charge_lineitems = CreditCardChargeLineItem.create_credit_card_charge_lineitems(expense_group, configuration)
```

Ruff (0.7.0)

`CreditCardChargeLineItem` may be undefined, or defined from star imports

(F405)